src/Program/Model/LuciConnection.hs

```diff
@@ -330,7 +330,7 @@ instance JS.LikeJS "Object" LuciScenario where
               (fromMaybe (JS.fromJSArray JS.emptyArray) (getProp "geometry_output" jsv >>= getProp "geometry")) t
   asJSVal (LuciResultScenario scId fc _) =
             setProp "ScID"  (JS.asJSVal scId)
-          $ setProp "FeatureCollection" fc newObj
+          $ setProp "GeometryInput" fc newObj
```

Comment:
This is not correct. You must not change the existing fields in luci messages. Especially this one - is a part of GeoJSON specification! Look at the [`siren` code](https://github.com/achirkin/qua-kit/blob/master/services/siren/sql/get_scenario.sql) for parsing scenario JSON:
```
[get_scenario message result]:
  - geometry_output:
    - format: "GeoJSON"
    - name
    - ScID
    - geometry:
      - type: FeatureCollection
      - features: [Features]
    - properties
  - ScID
  - created
  - lastmodified
```